### ISSUE TITLE
Use Kusto as the data source for 1ES and MMS images

### ISSUE THREAD

**Issue by dkurepa** [dotnet/arcade#9959]
We need to wait 30 days after https://github.com/dotnet/arcade/issues/9201 has been rolled out to populate the TimelineRecords Kusto table with Image Labels before letting MoT use it as it's source instead of doing log scraping itself

**Comment by dkurepa** [comment]
once this is merged, we could also introduce a new property in the Pipeline class, the number of times the pipeline was run in the last 30 days. This would be a breaking change to the models tho, and would require a version update

**Comment by dkurepa** [comment]
pr: https://dev.azure.com/dnceng/internal/_git/dotnet-helix-machines/pullrequest/24674

**Comment by dkurepa** [comment]
The pr has been merged. I will create a separate issue for adding Count to the outputs
